lib/ace/mode/yaml_highlight_rules.js

```diff
@@ -73,6 +73,22 @@ var YamlHighlightRules = function() {
             }, {
                 token : "string", // multi line string start
                 regex : '[|>][-+\\d\\s]*$',
+                onMatch: function(val, state, stack, line) {
+                    var indent = new RegExp(/^\s*(?:[\-?](?:$|\s))?/).exec(line)[0];
```

Comment:
It seems like Travis build didn't run for the last change due to the yesterday's API outage. The build with the change run successfully today in my account https://travis-ci.org/MaximTrushin/ace. Therefore I think the tests are fixed now.